README.md

```diff
@@ -82,12 +82,16 @@ parameters:
     stop_on_failure: false
     ignore_unstaged_changes: false
     hide_circumvention_tip: false
-    process_async_limit: 10
-    process_async_wait: 1000
     process_timeout: 60
     ascii:
         failed: grumphp-grumpy.txt
         succeeded: grumphp-happy.txt
+    parallel:
+        enabled: true
+        max_size: 32
```

Comment:
DX: Would suggest to rename this to max_threads.

Reply:
Gonna go for `max_workers`